tools/like_ps.py

```diff
@@ -65,12 +62,8 @@ def get_process_details(pid):
 
     data = {'user':'', 'cpu':0.0, 'mem':0.0, 'etime':'00:00', 'threads':0}
     try:
-        output = subprocess.check_output('ps o user,pcpu,pmem,etime,nlwp %i' % pid, shell=True)
-        try:
-            output = output.decode()
-        except AttributeError:
-            # Python2 catch
-            pass
+        output = subprocess.check_output(f"ps o user,pcpu,pmem,etime,nlwp {pid}", shell=True)
```

Comment:
Does this work if we use a:
```
output = subprocess.check_output(['ps', 'o', 'user,pcpu,pmem,etime,nlwp', str(pid)])
```
?

Reply:
See [f5b0b02](https://github.com/ledatelescope/bifrost/pull/216/commits/f5b0b02cb1cc8cd446a3f4d90cfc824276c56b64).